index.html

```diff
@@ -785,6 +788,51 @@
           "site preferences".
         </p>
       </section>
+      <section>
+        <h3>
+          “Nice to have” images
+        </h3>
+        <p>
+          Images are often content, but there are instances when an image is an
+          enhancement to the document, but not a necessary part of the core
+          content (a.k.a. “nice to have“). As such, Authors may choose to load
+          an image only above a certain breakpoint in order to
+          optimize the readbility of the core content at certain sizes by
+          reducing the visual clutter. This is a similar use case to <a href="#art-direction">Art direction</a>, but distinct enough to warrant separation.
```

Comment:
+1 to this being breakpoint-centric.

Here's yoav saying “it's art direction” at the meeting:
http://ircbot.responsiveimages.org/bot/log/respimg/2015-03-05#T117007

And in #56 Yoav specifically uses the word “breakpoints”.

This use case is about authors art-directing down to no image at all, and art direction means breakpoints.